src/exercises/01-Query-Renderer/README.md

```diff
@@ -314,6 +314,52 @@ We're already grabbing the artistID from the route params. We need to pass it in
 
 _./Artist1QueryRenderer.tsx_
 
+The last step for relay to understand which variable you are passing is to pass the variable next to Artist1QueryRendererQuery like this:
+
+```typescript
+     query Artist1QueryRendererQuery(**$artistID: ID!**){ 
+        artist(**id: $artistID**){
+           name
+           birthYear
+         }
+     }
+```
+
+This is how the whole component should look like in the end:
+
+``` typescript
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { Artist1 } from './Artist1';
+
+export const Artist1QueryRenderer = () => {
+  const { artistID } = useParams();
+
+  return (
+    <QueryRenderer<Artist1QueryRendererQuery>
+      environment={environment}
+      query={graphql`
+        query Artist1QueryRendererQuery($artistID: ID!) {
+          artist(id: $artistID) {
+            name
+            birthYear
+          }
+        }
+      `}
+      variables={{ artistID }}
+      render={({ props }) => {
+        if (!props || !props.artist) {
+          return <div>Loading</div>;
+        }
+        return <Artist1 artist={props.artist} />;
+      }}
+    />
+  );
+};
+
+```
```

Comment:
I've been avoiding putting the entire ending state in the instructions, because we also have it in the `completed` folder. Having the entire goal state in multiple places feels like an opportunity for drift in the docs -- the end state changes, and we remember to change it in the `completed` folder, but not here. 

Having said that, it's obvious that the end state was useful to you at this point in the docs -- what do you think about changing this bit to be a reminder that the entire final state is located in the completed folder if they want to see it?